### PR TITLE
Additional types

### PR DESCRIPTION
**Pull request by sashkopotapov**
This PR introduces new types, such as `Google Docs`, `Google Slides`, `Google Sheets`, `Affinity Photo`, `Affinity Design`, `Sketch`, and `Adobe Xd`.

### CODE REVIEW

File: Sources/UniformTypeIdentifier/PublicTypes.swift [hunk 1]
```diff
@@ -225,6 +228,31 @@ public extension UniformTypeIdentifier {
...
+}
+
+public extension UniformTypeIdentifier {
+    static let affinityPhoto = UniformTypeIdentifier(fileExtension: "afphoto")!
+    static let affinityDesign = UniformTypeIdentifier(fileExtension: "afdesign")!
+    static let sketchFile = UniformTypeIdentifier(
+        fileExtension: "sketch",
+        conformingTo: .zipArchive
+    )!
+    static let adobeXd = UniformTypeIdentifier(fileExtension: "xd", conformingTo: .zipArchive)!
```
Comment (sashkopotapov) [inline Sources/UniformTypeIdentifier/PublicTypes.swift:253]: They both has` application/zip` mime

**Comment by sashkopotapov** [comment]
@atdrendel, this one is ready for review.

File: Sources/UniformTypeIdentifier/PublicTypes.swift (hunk 1 above)
Comment (atdrendel) [reply Sources/UniformTypeIdentifier/PublicTypes.swift:253]: @sashkopotapov Can you add tests to make sure we can actually initialize the Affinity and Adobe UTIs?

File: Sources/UniformTypeIdentifier/PublicTypes.swift [hunk 2]
```diff
@@ -225,6 +228,31 @@ public extension UniformTypeIdentifier {
     static let oggApplication = UniformTypeIdentifier(mimeType: "application/ogg")!
 }
 
+public extension UniformTypeIdentifier {
+    static let googleDocument = UniformTypeIdentifier(
+        mimeType: "application/vnd.google-apps.document",
+        conformingTo: .compositeContent
```
Comment (atdrendel) [inline Sources/UniformTypeIdentifier/PublicTypes.swift:234]: `compositeContent`. That's interesting. Makes sense, though.

**Review by atdrendel — APPROVED** [review]
I'd like you to add a test for the types you initialize via file extension just to make sure we can actually create them successfully. Other than that, this is good to go in.

**Pull request merged**